draft-seemann-quic-address-discovery.md

```diff
@@ -169,13 +171,25 @@ possible, define some validation logic (e.g. by asking multiple untrusted peers
 and observing if the responses are consistent). This logic is out of scope for
 this document.
 
-## On the Responder Side
+## On the Responder Side {#responder-side-security}
 
 Depending on the routing setup, a node might not be able to observe the peer's
 reflexive transport address, and attempts to do so might reveal details about
 the internal network. In these cases, the node SHOULD NOT offer to provide
 address observations.
 
+On path attackers could capture "normal" packets sent from requester to
+responder, and resend them from a spoofed source address. Such spoofed
+packets could trigger production of spurious OBSERVED_ADDRESS frames.
+The recommendation to only include OBSERVED_ADDRESS frames in packets
+sent on the same path over which the address was observed offers a first
+layer of protection, as these packets will not reach the requester if
```

Comment:
How does this protect against the sender?

Reply:
If sent over the same path, they will only reach the peer if the path is valid, thus will not have an effect on the peer. So, no pollution. But yes, still a volume effect. Maybe that could be said differently